.nvmrc

```diff
@@ -0,0 +1 @@
+v8.17.0
```

Comment:
Can we agree on a version? The docker image in `RP` https://github.com/antistatique/retraitespopulaires/blob/dev/Dockerfile#L1 is on `node-10`

Reply:
the styleguide and the Embeed styleguide are not the same, the node version are then note the same